pallets/social/src/reactions.rs

```diff
@@ -0,0 +1,20 @@
+use super::*;
+
+impl<T: Trait> Module<T> {
+
+    // FIXME: don't add reaction in storage before the checks in 'create_reaction' are done
+    pub fn insert_new_reaction(account: T::AccountId, kind: ReactionKind) -> ReactionId {
```

Comment:
Maybe move it to `impl<T: Trait> Reaction<T> {}`?

Makes sense due to we use that principle with `Space`/`Post` for expample.